src/utils/blockchain/contracts.testnet.ts

```diff
@@ -333,4 +334,12 @@ export const contracts = {
     abi: marginTokenAbi,
     chainId: ChainId.BSC_TESTNET,
   },
+  ZUSD_token: {
+    address: '0x4a0741FA749eD6b1F810224D09f1f511952e67de',
+    abi: ZUSDAbi,
```

Comment:
At this point dapp only supports ERC20 capabilities for ZUSD token, so you can just reuse erc20TokenAbi